### PR TITLE
Gut tests update

### LINKED ISSUE #213: Update GUT Test to work with last Rakugo verison
GUT Tests doesn't work now as they use old signal names and old funcs names.

### PR DESCRIPTION
**Pull request by Jeremi360**
this fixes #213

### CODE REVIEW

File: Test/TestGui/TestGui.gd [hunk 1]
```diff
@@ -7,18 +7,24 @@ const file_name = "res://Test/TestParser/Timeline.rk"
 @onready var menu_container = $MenuContainer
 
 func _ready():
-	Rakugo.connect("say",Callable(self,"_on_say"))
-	Rakugo.connect("step",Callable(self,"_on_step"))
-	Rakugo.connect("ask",Callable(self,"_on_ask"))
-	Rakugo.connect("menu",Callable(self,"_on_menu"))
-	Rakugo.connect("notify",Callable(self,"_on_notify"))
+	Rakugo.add_custom_regex("HW", "^hello_world$")
+	Rakugo.connect("sg_say",Callable(self,"_on_say"))
```
Comment (theludovyc) [inline Test/TestGui/TestGui.gd]: If you use Godot V4 signal you can write
Rakugo.sg_say.connect(_on_say)

https://docs.godotengine.org/en/stable/getting_started/step_by_step/signals.html#complete-script

**Review by theludovyc — CHANGES_REQUESTED** [review]

File: Test/TestGui/TestGui.gd (hunk 1 above)
Comment (Jeremi360) [reply Test/TestGui/TestGui.gd]: Okey I will update those calls.

Comment (Jeremi360) [reply Test/TestGui/TestGui.gd]: Just fixed with my last commit @theludovyc

**Review by theludovyc — APPROVED** [review]

**Comment by theludovyc** [comment]
Ready for merge :) !

**Pull request merged**